src/index.ts

```diff
@@ -93,18 +116,35 @@ You can also use special values to control access to orgs:
     },
   ];
 
+  private telemetry?: Telemetry;
+
   public async run(): Promise<void> {
     const { flags } = await this.parse(McpServerCommand);
+    if (!flags['no-telemetry']) {
+      this.telemetry = new Telemetry(this.config);
+      await this.telemetry.start({
+        toolsets: flags.toolsets.join(', '),
+        orgs: sanitizeOrgInput(flags.orgs),
+      });
+
+      process.stdin.on('close', () => {
+        this.telemetry?.stop();
+      });
```

Comment:
should there be a separate event to differentiate from "successful server exit" and "server crashed" (exit code >= 1)